### PR TITLE
[Javascript] - Extract concepts of v2 pangram exercise

### PR DESCRIPTION
**Pull request by hayashi-ay**
#653

### CODE REVIEW

File: languages/javascript/reference/exercise-concepts/pangram.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
+# Concepts for Pangram
+
+## Approaches
+There are two approaches for solving Pangram so I divided concepts into common parts and solution unique parts.
```
Comment (SleeplessByte) [inline languages/javascript/reference/exercise-concepts/pangram.md:4]:
```suggestion

There are two approaches for solving Pangram so I divided concepts into common parts and solution unique parts.
```

File: languages/javascript/reference/exercise-concepts/pangram.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+## Common Concepts
+ - String#toLowerCase(or String#toUpperCase) used for normalising the iput
```
Comment (SleeplessByte) [inline languages/javascript/reference/exercise-concepts/pangram.md:7]:
```suggestion

- String#toLowerCase(or String#toUpperCase) used for normalising the iput
```

File: languages/javascript/reference/exercise-concepts/pangram.md (hunk 1 above)
Comment (hayashi-ay) [reply languages/javascript/reference/exercise-concepts/pangram.md:5]: I fix it.

File: languages/javascript/reference/exercise-concepts/pangram.md (hunk 2 above)
Comment (hayashi-ay) [reply languages/javascript/reference/exercise-concepts/pangram.md:9]: same.

**Comment by hayashi-ay** [comment]
I fix the conflict.

**Comment by SleeplessByte** [comment]
Thank you 🎈

**Pull request merged**